### ISSUE TITLE
Code Highlighting does not work

### ISSUE THREAD

**Issue by rezpe** [slara/generator-reveal#69]
I have seen that code highlighting does not work. If you create a new slide:

    yo reveal:slide "Example Slide"

and then you put this in the slide as a test:

``` html
<pre><code class="javascript">
var a = 34
var b = function(){
return "great"
}
</code></pre>

<pre><code>
System.out.println("What is this?");
String p = "this is p";
</code></pre>
```

You' ll see the code won't be highlighted.

I believe the reason for this is that the code will be highlighted only if already present from the start in the DOM of index.htm. However, as the html slides are loaded one by one and not present in the DOM, the code won't be highlighted.

**Comment by janraasch** [comment]
Hi @rezpe,

thank you for the detailed description. You are right. You need to remove the condition

```javascript
condition: function() { return !!document.querySelector( 'pre code' ); }
```

from line `77` of your generated `templates/_index.html` file.

I will update this to a more sane default so highlighting works out of the box.

**Comment by rezpe** [comment]
Thanks Jan !! Let us know when to update the generator to have this :)

**Comment by janraasch** [comment]
As I said, you may simply update your locally generated `templates/_index.html`. Plus, the fix is now live with `v0.5.2`.
